README.md

```diff
@@ -27,6 +33,20 @@ If you're one of the brave few who wants to hack the firmware then read on.
 
 Going forward, it's easier to flash the firmware of your choice by using the downwards toolbar icon which is located rightwards of the *green play + toolbox icon*.
 
+### Minimal development setup
+
+1. Install the ARM cross-compiler, cross-assembler and stdlib implementation. Eg. on Arch Linux the packages `arm-none-eabi-binutils`, `arm-none-eabi-gcc`, `arm-none-eabi-newlib`.
+
+2. Install Node.js v12. If you have a later version, editing the version requirement in `lib/agent/package.json` *might* work.
```

Comment:
It is a very interest point. The agent built on the electron and every electron has it own node.js runtime. We always use the same version for development that provided by the electron.
The repo contains an `.nvmrc` file that contains the recommended node.js version for development. It is a de facto standard in the node ecosystem.
To manage multiple node.js on 1 machine recommended to use the [nvm](https://github.com/nvm-sh/nvm) or windows equivalent.

There are many technical requirement of an electron major version upgrade. For example we have to wait to all of our 3rd party dependencies support it.
So I don't recommend to use other version for development that we use.
This is the reason, why we implemented the node.js version checking as a prerequisite of the build.

Reply:
Yeah, I expected as much. For just running the flashing scripts the hacky approach seems sufficient, but I wouldn't do that to actually build the Agent for use.